app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -217,8 +217,17 @@ def systemtypes
   end
 
   def cloud_volume_types
-    # get only the active storage
-    collector.storage_types.each do |v|
+    # TODO: The Power Cloud API does not yet have a call to retrieve
+    # available storage types. When availabile this hardcoded list should
+    # be removed and the list should come from collector.storage_types
+    all_possible_storage_types =
```

Comment:
@jaywcarman could you return this list from the [collector](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/67/files#diff-53f571cf57b42afefdd9b8a927cff8beR47) so the parser can stay the same when we switch over to using an API call?

Reply:
@agrare I can definitely do that, but full disclosure: I have not been able to get a guarantee that this data structure will stay intact.